src/BuildApp.jl

```diff
@@ -0,0 +1,227 @@
+module BuildApp
+
+using Glob, PackageCompiler
+
+include("sign_mac_app.jl")
+
+function build_app_bundle(parsed_args)
```

Comment:
I was wondering if `build_app_bundle` could be of a similar API as described in #7. Since the user is primarily going to be interacting with this function, perhaps it would be nice if he can specify an array of paths to libraries and array of paths to resources. What do you think? :-)

Reply:
Ah, yeah, let me make the `resources` and `libraries` args a bit nicer, like the way you have them.

Otherwise, I think the API looks like you have it in #7 now? Yes, the goal of this PR was exactly that, to make the API match what you had in #7 so that the windows and mac versions could be the same, since I agree the API as you have it is the nicest! :)